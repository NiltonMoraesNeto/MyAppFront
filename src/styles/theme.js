export const theme = {
  colors: {
    primaryColor: '#0A1128',
    secundaryColor: '#dc143c',
    white: '#FFFFFF',
    lightBg: '#FDF5E6',
    darkBg: '#0A1128',
  },
  fonts: {},
  spacings: {},
};
