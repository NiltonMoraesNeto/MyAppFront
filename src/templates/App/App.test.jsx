import Home from '.';
import { renderTheme } from '../../styles/render-theme';
import { screen } from '@testing-library/react';
import { theme } from '../../styles/theme';

test('renders learn react link', () => {
  const { debug } = renderTheme(<Home />);
  const headingContainer = screen.getByRole('heading', {
    name: 'TEXTO TESTE',
  }).parentElement;
  expect(headingContainer).toHaveStyle({
    //background: 'blue',
    background: theme.colors.primaryColor,
  });
  expect(headingContainer).toMatchSnapshot();
  expect(headingContainer).toHaveStyleRule('background', '#0A1128');
});
