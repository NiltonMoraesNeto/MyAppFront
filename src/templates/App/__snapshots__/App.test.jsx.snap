// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
.c1 {
  color: #0A1128;
}

.c0 {
  background: #0A1128;
}

<div
  class="c0"
>
  <h1
    class="c1"
  >
    TEXTO TESTE
  </h1>
</div>
`;
